deps/spidershim/src/v8v8.cc

```diff
@@ -35,7 +35,7 @@ bool gDisposed = false;
 bool V8::Initialize() {
   assert(!internal::gDisposed);
   return v8::internal::InitializeIsolate() &&
-         v8::internal::InitializeHandleScope() && JS_Init();
+         v8::internal::InitializeHandleScope() /* && JS_Init() */;
```

Comment:
OK, I have landed a JS_IsInitialized() API which lets us fix this properly.  Will do.  Filed #181.